src/components/pages/InstructorHome/InstructorSidebar.js

```diff
@@ -0,0 +1,44 @@
+import { Layout, Menu, Breadcrumb } from 'antd';
+import React from 'react';
+import {
+  DesktopOutlined,
+  PieChartOutlined,
+  FileOutlined,
+  TeamOutlined,
+  UserOutlined,
+} from '@ant-design/icons';
+
+const { SubMenu } = Menu;
+const { Sider } = Layout;
+
+function InstructorSidebar() {
+  return (
+    <Layout style={{ minHeight: '100vh' }}>
+      <Sider>
+        <div className="logo" />
+        <Menu theme="dark" defaultSelectedKeys={['1']} mode="inline">
```

Comment:
I see that you used the default selected keys to preselect the menu item that should be shown first. Instead of using nums, could we use the title of the menu-item as a key? Titles could be more intuitive.

Reply:
Makes sense, I'm assuming your comment is referring to changing the key's to match the text inside of the `Menu.Items`, I'm thinking:

- Option 1 text becomes - > Dashboard
- Key for Dashboard becomes "dashboard" instead of "1"
- Pre-select based on "dashboard" instead of the number 1?

Hopefully this isn't confusing, just wanting to make sure we're on the same page.

Reply:
Yes, the key for each menu item can be changed to the title of the item. We haven't decided what kind of information to put in the sidebar. So I was thinking to leave it like that until I hear from the UX and then I can change the key and also the name of each item. However, if you want me to change it today, I can do it later.

Reply:
That makes sense Jie. I'm good with leaving it nums until we decide exactly what these items are. Good call out Joey. We should try to figure out what the menu items will be.

Reply:
These are some fillers that you can use for the navbar. What do you think @ashtilawat23 ?

Sidebar Links:
- Overview
- Class
- Messages